talisker/revision.py

```diff
@@ -56,7 +56,8 @@ def bzr_version_info():
 
 
 def setup_py():
-    return subprocess.check_output([sys.executable, 'setup.py', '--version'])
+    return subprocess.check_output(
+        [sys.executable, 'setup.py', '--version'], stderr=subprocess.STDOUT)
```

Comment:
Hmm, that was my thinking, but doesn't actuall make sense, as the output will be lost anyway.